controllers/allcontrollers.go

```diff
@@ -36,6 +41,38 @@ const (
 // one per resource type
 var appInformers = make(map[string]informers.GenericInformer)
 
+// broadcastEvent broadcasts an event to the controller
+func broadcastEvent(eventtype, reason, message string, client k8sclient.Interface) {
+	ns, ok := os.LookupEnv(podNamespaceEnvVariable)
+	if !ok {
+		log.Logger.Errorf("could not get pod namespace from environment variable %s", podNamespaceEnvVariable)
```

Comment:
I'm inclined to think these should be info level statements. warning at most. if we fail to issue an event, I wouldn't call it an error.